### PR TITLE
Apollo

### PR DESCRIPTION
**Pull request by khrystyna-sb**

### CODE REVIEW

File: Countries/Queries.graphql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+query CountriesAPI {
+    countries {
+        code
+        name
+        capital
+        languages {
+            name
+                native
+                }
+        }
```
Comment (ambrusha) [inline Countries/Queries.graphql]: alignment

File: Countries/Apollo.swift [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+class Apollo {
+    static let shared = Apollo()
+    
+    let client: ApolloClient
+    // 3
+    init() {
+        client = ApolloClient(url: URL(string: "https://countries.trevorblades.com/")!)
+    }
+}
```
Comment (ambrusha) [inline Countries/Apollo.swift]: Looks like almost `Singleton` may be it can be more obvious?

File: Countries/Country.swift [hunk 3]
```diff
@@ -7,15 +7,15 @@
...
+//    var name: String
+//    var capital: String
+//    var continent: Continent
+//    var emoji: String
+//}
+//
+//struct Continent: Decodable {
+//    var name: String
+////    var countries: [Country]
+//}
```
Comment (ambrusha) [inline Countries/Country.swift]: Should it file still be in code base if it all commented?

File: Countries/CountryGenerator.swift [hunk 4]
```diff
@@ -7,6 +7,6 @@
 
 import Foundation
 
-class CountryGenerator{
-    let country1 = Country(name: "Norway", capital: "Oslo", continent: Continent(name: "Europe"), emoji: "🇳🇴")
-}
+//class CountryGenerator{
+//    let country1 = Country(name: "Norway", capital: "Oslo", continent: Continent(name: "Europe"), emoji: "🇳🇴")
+//}
```
Comment (ambrusha) [inline Countries/CountryGenerator.swift]: saa

File: Countries/Country.swift (hunk 3 above)
Comment (khrystyna-sb) [reply Countries/Country.swift]: Ok, you're right I will delete it now)

File: Countries/Apollo.swift [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+import Apollo
+
+class Apollo {
+    
+    static let shared = Apollo()
+    
+    let client: ApolloClient
+
+    private init() {
+        client = ApolloClient(url: URL(string: "https://countries.trevorblades.com/")!)
```
Comment (ambrusha) [inline Countries/Apollo.swift]: What you will do when API will become unavailable?
I mean FORCE UNWRAP

Comment (khrystyna-sb) [reply Countries/Apollo.swift]: oh no! how could I miss it!?
thank you

**Review by ambrusha — APPROVED** [review]

**Pull request merged**